cmd/registry/cmd/apply/apply_test.go

```diff
@@ -78,7 +78,7 @@ func TestApply(t *testing.T) {
 		}
 		api := project.Api("registry")
 		// TODO: Prefer using the API client directly instead of relying on application code that isn't directly related to this test.
-		_, err = core.GetAPI(ctx, registryClient, api, func(message *rpc.Api) {
+		err = core.GetAPI(ctx, registryClient, api, func(message *rpc.Api) {
```

Comment:
Not a blocker, and I've done this too - but this change (overall in this file) seems big for this PR.